library/selinux_el7.py

```
@@ -0,0 +1,393 @@
+        return None
+    return all_enabled
+
+
+# setter subroutines
+def set_config_state(module, state, configfile):
+    # SELINUX=permissive
+    # edit config file with state value
+    stateline = "SELINUX=%s" % state
+    lines = get_file_lines(configfile, strip=False)
+
+    tmpfd, tmpfile = tempfile.mkstemp()
```

Ruff (0.16.1)

[warning] Unpacked variable `tmpfd` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)